websites/management/commands/sync_contents_with_config.py

```diff
@@ -0,0 +1,76 @@
+"""Updates derived values in WebsiteContent records to match the site config"""
+
+from django.core.management import BaseCommand
+
+from websites.models import Website, WebsiteContent, WebsiteStarter
+from websites.site_config_api import config_item_iter, has_file_target, is_page_content
+
+
+class Command(BaseCommand):
+    """Updates derived values in WebsiteContent records to match the site config"""
+
+    help = __doc__
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "-s",
+            "--starter-slug",
+            dest="starter_slug",
+            default="",
+            help="If specified, only sync contents for websites that user the given starter.",
+        )
+
+    def handle(self, *args, **options):
+        starter_qset = WebsiteStarter.objects.all()
+        if "starter_slug" in options:
```

Comment:
I think we need to check if `options["starter_slug"]` is empty, since the value will always be set. If it's missing the default value of `""` is used but it's still present in the dict